PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## Description
+<!--
+Tell us what your PR does.
+Please attach a screenshot/ video/gif image describing your PR if possible.
+-->
+
+## Issue fixed
+<!--
+Please list out all issue fixed with this PR here.
+-->
+
+<!--
+Please make sure you fill in these checkboxes,
+your PR will be reviewed faster if we know exactly what it does.
+-->
```

Comment:
@ZeroX-DG can you add the following text?
```
Put an `x` in all the boxes that apply.
```

Reply:
@daiyam Thanks for the feedback, I've updated the message. Do you have any other idea?